sudoku/src/main/java/de/sfuhrm/sudoku/output/PlainTextFormatter.java

```diff
@@ -43,6 +43,9 @@ public String format(final GameMatrix matrix) {
                     str = Integer.toString(val + 1 - schema.getMinimumValue());
                 }
                 sb.append(str);
+                if (column < schema.getWidth() - 1) {
```

Comment:
"1" has a different width than "15" for 16x16 sudokus.

I suggest to fill the current column (sb.append(str)) with spaces until we have 
`Integer.toString(schema.getMaximumValue()).length()` characters for each cell.
This way we get a beautiful ASCII table.

Reply:
good suggestion, but i would make that optional since single spaces are more trivial to parse and those extra spaces are only for human readability

Reply:
Feel free to do so if you like.